src/test/java/lt/vu/trip/service/user/TripServiceImplDataProvider.java

```diff
@@ -0,0 +1,102 @@
+package lt.vu.trip.service.user;
```

Comment:
This class is only a data provider for tests. Here are a lot of cases with different parameters/filters and expected results.